enhancements/network-observability/network-observability.md

```diff
@@ -127,92 +135,120 @@ However, the first release will not analyze why something went wrong.
-    SDN and the majority of customers run OpenShift SDN today.
-    Therefore in a brown field scenario, this would require the user to change
-    their CNI type before network observability can be enabled.
-        Note that the effort to implement OpenShift SDN to configure OVS is
-    doable.  However, new features recently added have only been implemented
-    in OVN-Kubernetes.
-<br>
-    - Kernel must be 4.18+ with eBPF enabled
+    - With `EBPF` agent, Kernel must be 4.18+ with eBPF enabled
         The eBPF agent will either need `BPF`, `PERFMON`, `NET_ADMIN`, `SYS_RESOURCE` 
     Linux capabilities or Administrative privileges if the kernel doesn't 
     recognize `BPF` and `PERFMON` (eg K3s / Kind).
 
+    - With `IPFIX` agent, CNI must be OVN-Kubernetes
+        The network type (CNI) has to be OVN-Kubernetes since configuring OVS to
+    export IPFIX data is only supported there.
+<br>
+
 2. Data sampling<br>
     In order to scale in terms of collecting data, it is not uncommon to take
-a very small sample of data. By default, the data sample rate in OVS is
-400 to 1.  Networks with larger bandwidth, sample at an even higher rate
-such as 1,024 to 1.
+a very small sample of data. By default, the data sample rate with agent `EBPF` is
+50 to 1. Users may have to tweak this value until they get an acceptable tradeof between
+observability and resources consumption overhead.
 
-    With sampling, you will be able to get trends about your network.  If
+    With sampling, you will be able to get trends about your network. If
 you are looking for a specific flow of data, it's likely not to be there.
 The total number of bytes and packets are just estimations when sampling
-is used.  Therefore if you need full visibility of the traffic and accuracy,
+is used. Therefore if you need full visibility of the traffic and accuracy,
 the only choice is not do sampling.
 
 3. Resources required<br>
     Exporting data is potentially CPU, memory and bandwidth-intensive, and
-typically requires a large amount of storage.  The CPU/memory/bandwidth
-resources can be managed by the sampling data rate.  The storage requirements
+typically requires a large amount of storage. The CPU/memory/bandwidth
+resources can be managed by the sampling data rate. The storage requirements
 can be controlled by the data retention policy or how long to keep the data.
+Other parameters can be configured to impact the resources consumption, such
+as Kafka and Loki client batch settings (size, timeouts...).
+
+    For Loki and Kafka, other settings are on the server side (e.g. Loki rate
+limits).
 
     Nevertheless, in order to maintain the same performance that you have
 prior to enabling network observability, will require additional resources.
 The specific number of cores and the amount of memory and storage required
 will be finalized as more testing is done.
 
-### CRD
+#### New metrics
```

Comment:
@simonpasquier this might be something to review for you or your team, I guess

Reply:
Thanks for the heads-up! As you noted, the main concern comes from metrics with potentially high cardinality and the impact on platform monitoring.
* Is there any mechanism planned to turn off metrics collection or reduce the amount of metrics in case platform monitoring gets overloaded?
* It would also be interesting to provide a rough estimate of how many new series this could generate (e.g. "with X nodes and Y pods, we expect Z series").